gravity-dex.md

```diff
@@ -60,39 +60,40 @@
 - [Discord channel](https://discord.gg/vcExX9T)
 - [Peers and seed nodes](https://hackmd.io/@KFEZk8oMTz6vBlwADz0M4A/BkKEUOsZu#)
 - [Performance Testing for Liquidity module(Gravity DEX)](https://github.com/tendermint/liquidity/blob/develop/doc/Performance%20Testing%20for%20Liquidity%20Module.pdf)
+
 ### Hardware Requirements
 
 Minimum requirements
 
 - 2 Core CPU
-- 8GB RAM
+- 8 GB RAM
```

Comment:
I did not know we need space between the number and gigabytes. Is this conventional?